src/mlpack/methods/naive_bayes/nbc_main.cpp

```diff
@@ -51,9 +51,9 @@ PROGRAM_INFO("Parametric Naive Bayes Classifier",
     "cases."
     "\n\n"
     "If classifying a test set is desired, the test set may be specified with "
-    "the " + PRINT_PARAM_STRING("test") + " parameter, and the "
-    "classifications may be saved with the " + PRINT_PARAM_STRING("output") +
-    " output parameter.  If saving the trained model is desired, this may be "
+    "the " + PRINT_PARAM_STRING("test") + " parameter, and the classifications"
+    " may be saved with the " + PRINT_PARAM_STRING("predictions") +"predictions"
+    "  parameter.  If saving the trained model is desired, this may be "
     "done with the " + PRINT_PARAM_STRING("output_model") + " output "
     "parameter."
     "\n\n"
```

Comment:
Can we add a paragraph pointing out that the `output` (and `output_probs` when you do it) options are deprecated and will be removed in mlpack 4.0.0?  You can take a look at `logistic_regression_main.cpp` to get an idea of what I mean.

Reply:
@rcurtin i have a question so wouldn't the changes i made be visible while i use mlpack_nbc --h ?

Reply:
Yes, when you recompile, the changes you make here should be visible when you run `mlpack_nbc -h`.